korio/src/commonMain/kotlin/com/soywiz/korio/async/ObservableProperty.kt

```diff
@@ -0,0 +1,22 @@
+package com.soywiz.korio.async
+
+import com.soywiz.kds.iterators.fastForEach
+
+class ObservableProperty<T>(default: T) {
```

Comment:
Minor: what do you think about renaming `default` to `initial`?

Reply:
Oh, I thought about this, but forgot to rename it, thanks

Reply:
Done